### PR TITLE
chore: updated admin routes, login and register

### PR DESCRIPTION
**Pull request by karlk08**
updated routes for admin login, register and admin pages to ensure unauthorized user doesn't have access to the admin page. making sure to reroute unauthorized user to the login page if not logged in.

### CODE REVIEW

File: src/Admin/admin-authContext/AuthContext.tsx
Comment (robertbolgeo) [inline src/Admin/admin-authContext/AuthContext.tsx:1]: This looks good, but can you explain why the condition to throw an error in line 140 is necessary?
I understand how the useContext is working in Auth, but this hook and return are a little difficult for me.

File: src/App.tsx
Comment (robertbolgeo) [inline src/App.tsx:1]: shouldn't </AdminProtectedRoutes> go at the end of the entire route? Or does putting it around the layout make everything in the layout receive the protection?

**Review by robertbolgeo — APPROVED** [review]
I think this looks good, but please also see the comments.

**Review by susheepls — APPROVED** [review]
Looks good to me but what are the children that adminprotectedRoute is returning?

**Comment by karlk08** [comment]
> Looks good to me but what are the children that adminprotectedRoute is returning?

if user is logged in it will return every content contained inside the private or protected route "children" aka admin pages.

File: src/Admin/admin-authContext/AuthContext.tsx
Comment (karlk08) [reply src/Admin/admin-authContext/AuthContext.tsx:1]: for error handling, to make sure that useAuth hook retrieves the authentication from AuthContext

File: src/App.tsx
Comment (karlk08) [reply src/App.tsx:1]: If you take a look at the route line 65 and  72 it closes all admin page within the protected route

**Pull request merged**